src/frontend/packages/lib_video/src/components/common/DashboardControlPane/index.tsx

```diff
@@ -63,11 +74,21 @@ export const DashboardControlPane = ({ isLive }: DashboarControlPaneParams) => {
           <Tab title={intl.formatMessage(messages.titleConfiguration)}>
             <VideoWidgetProvider isLive={isLive} isTeacher />
           </Tab>
-          <Tab title={intl.formatMessage(messages.titleAttendance)}>
-            <DashboardLiveTabAttendance />
-          </Tab>
+          {hasStats ? (
+            <Tab title={intl.formatMessage(messages.titleStats)}>
+              <DashboardTabStatistics />
+            </Tab>
+          ) : (
+            <Tab title={intl.formatMessage(messages.titleAttendance)}>
+              <DashboardLiveTabAttendance />
+            </Tab>
+          )}
         </Tabs>
       </ThemeContext.Extend>
     </Box>
   );
 };
+
+DashboardControlPane.defaultProps = {
+  hasStats: false,
+};
```

Comment:
If you want to stick to the project convention you should not define defaultProps but rather setting the prop optional and set a default value in the function component declaration